app/tut/ga/creatlach.md

```diff
@@ -0,0 +1,967 @@
+[[le hoidhreacht|through inheritance]] ó gcreatlach eile. Cad is brí le sin? Is féidir linn
+creatlach [["tuismitheoir"|parent]] a chruthú, agus roinnt creatlacha [["páiste"|child]]; Ansin
+beidh gach gníomh creatlaí atá ag an **[[máthair-creatlach|parent-outline]]** ag gach
+**[[mac-creatlach|child-outline]]** freisin. (Is féidir linn [["for-creatlach"|super-outline]] agus
+"[[fo-creatlach|sub-outline]]" a thabhairt ar na gcreatlacha sin freisin.)
+
+Tá sé a lán níos soiléire le sampla:
+
+## Sampla
+
+Féach anois ar an gcreatlach seo:
+
+```{.setanta .numberLines}
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+```
+
+Níl ach gníomh amháin aige (`bog`) agus nuair a ghlaoitear air, scríobhann sé "Táim ag bogadh" ar an
+gconsól.
+
+Anois cruthaímis creatlach nua `Madra` atá mac-creatlach do `Ainmhí`. Úsáidimid an
+eochairfhocail [[`ó`{.setanta}|from]] chun é sin a dhéanamh.
+
+```{.setanta .numberLines}
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+}
+```
+
+Níl aon iompraíocht ag `Madra` go díreach, ach toisc go bhfuil sé ina mhac d'`Ainmhí`, faigheann sé
+an gníomh `bog` le hoidhreacht. Chun é sin a fheiceáil cruthaímis ásc `Madra` agus glaoimid ar `bog`
+leis.
+
+```{.setanta .numberLines}
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+}
+
+m := Madra()
+bog@m()
+```
+
+Bain triail as:
+
+{{{
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+}
+
+m := Madra()
+bog@m()
+}}}
+
+Ba chóir gur chonaic tú "Táim ag bogadh" ar an gconsól.
+
+Is féidir linn gníomhartha a chuir sa chreatlach `Madra` go díreach freisin, mar seo:
+
+```{.setanta .numberLines}
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+    gníomh ith() {
+        scríobh("Is aoibhinn liom ispíní")
+    }
+}
+
+m := Madra()
+bog@m()
+```
+
+Anois scríobhann `ith@m()` "Is aoibhinn liom ispíní" ar an gconsól. Táimid in ann mac-creatlach eile
+a chruthú ó `Ainmhí` arís:
+
+```{.setanta .numberLines}
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+}
+
+creatlach Cat ó Ainmhí {
+}
+
+m := Madra()
+bog@m()
+
+c := Cat()
+bog@c()
+```
+
+Tá an gníomh `bog` ag an gcat freisin, bain triail as an gcód:
+
+{{{
+creatlach Ainmhí {
+    gníomh bog() {
+        scríobh("Táim ag bogadh")
+    }
+}
+
+creatlach Madra ó Ainmhí {
+}
+
+creatlach Cat ó Ainmhí {
+}
+
+m := Madra()
+bog@m()
+
+c := Cat()
+bog@c()
+}}}
+
+Is féidir linn oidhreacht a úsáid chun gníomhartha a roinnt le go leor creatlacha, nach bhfuil sé
+sin an-úsáideach!
+
+## `tuis`{.setanta}
+
+Tá eochairfhocail eile ag *Setanta*, `tuis`{.setanta}. Is féidir linn `tuis`{.setanta} a úsáid chun
```

Comment:
ma tá tuis[mitheoir] in úsáid, b'fhéidir gur cheart tuismitheoir a úsáid tríd síos?